src/shared/components/results/ResultsFacets.tsx

```diff
@@ -1,50 +1,36 @@
-import { FC, ReactElement, useMemo } from 'react';
-import { Link, useRouteMatch } from 'react-router-dom';
-import { Facets } from 'franklin-sites';
+import { FC } from 'react';
+import { Facets, Facet } from 'franklin-sites';
 
-import { Facet } from '../../../uniprotkb/types/responseTypes';
+import TaxonomyFacet from './TaxonomyFacet';
 
-import { Location, LocationToPath } from '../../../app/config/urls';
+import useNS from '../../hooks/useNS';
+
+import { FacetObject } from '../../../uniprotkb/types/responseTypes';
 
 import './styles/results-view.scss';
 
-const ResultsFacets: FC<{ facets: Facet[]; isStale?: boolean }> = ({
+const ResultsFacets: FC<{ facets: FacetObject[]; isStale?: boolean }> = ({
   facets,
   isStale,
 }) => {
-  const match = useRouteMatch<{ subPage?: string }>(
-    LocationToPath[Location.BlastResult]
+  const isSearchPage = Boolean(useNS());
+
+  const splitIndex = facets.findIndex(
+    (facet) => facet.name === 'model_organism'
```

Comment:
Would be good to have facet names as an enum

Reply:
I think that deserves a TODO.
Because we have a limited set of possible facets available on the server, depending on the endpoint/namespace. But we also have a subset of those facets that are the ones we actually query the server explicitely.
I think these need to be tied somehow (see the defaultFacets variable in the apiUrls.ts file).
So a change taking into account everything would be quite a major refactor don't you think?